### PR TITLE
Pass ev docs through the grammarly

### PR DESCRIPTION
**Pull request by pepe**
I passed the whole text through Grammarly with these settings:
Grammarly settings.

I also added the last section to warn but also calm. I hope anyway :-).

### CODE REVIEW

File: content/docs/event_loop.mdz [hunk 1]
```diff
@@ -3,168 +3,217 @@
...
+out of the box - the event loop. The event loop provides concurrency within
+a single thread by allowing cooperating fibers to yield instead of blocking
+forward progress. An event loop is a form of cooperative multi-tasking that
+can be useful in many applications, like situations where there are many
+concurrent, IO-bound tasks. So tasks that spend most of their execution time
+waiting for network or disk, rather than using CPU time.
 
-Most event loop functionality in Janet is exposed through the @code`ev/` module, but many other functions outside this module
-will interact with the event loop.
+Most event loop functionality in Janet exposes through the @code`ev/` module,
```
Comment (pyrmont) [inline content/docs/event_loop.mdz:14]: This is an awkward result from a poor heuristic that says that the passive voice is always bad.

File: content/docs/event_loop.mdz [hunk 2]
```diff
@@ -3,168 +3,217 @@
...
-
-# Start bob working in a new task with ev/call
+  "Does some work."
+  [name n]
+  (for i 0 n
+    (print name " working " i "...")
+    (ev/sleep 0.5))
+  (print name " is done!"))
+
+# Start bob working on a new task with ev/call
```
Comment (pyrmont) [inline content/docs/event_loop.mdz:85]: This is wrong in this context.

File: content/docs/event_loop.mdz [hunk 3]
```diff
@@ -3,168 +3,217 @@
...
 (ev/call worker "bob" 10)
 
 (ev/sleep 0.25)
 
-# Start sally working in a new task with ev/go
+# Start sally working on a new task with ev/go
```
Comment (pyrmont) [inline content/docs/event_loop.mdz:90]: This is wrong in this context.

**Comment by pepe** [comment]
One more thing comes to mind, and it is this doc order. Would you consider putting it just after fibers (and rewrite the intro a little)?

File: content/docs/event_loop.mdz [hunk 4]
```diff
@@ -3,168 +3,217 @@
...
+the newly created task fiber immediately.
 
-The more general way to start new tasks is @code`ev/go`, which takes as an argument a fiber to be resumed by the event loop scheduler.
-@code`ev/go` allows the programmer to customize fiber flags as well as specify a supervisor channel for the newly created task.
+The more general way to start new tasks is @code`ev/go`, which takes as an
+argument a fiber to be resumed by the event loop scheduler.
 
-Lastly, there is the @code`ev/spawn` macro for concisely running a series of forms in a new task.
+The @code`ev/go` allows the programmer to customize fiber flags and specify a
+supervisor channel for the newly created task.
```
Comment (pyrmont) [inline content/docs/event_loop.mdz:107]: This is a slight improvement.

File: content/docs/event_loop.mdz [hunk 5]
```diff
@@ -3,168 +3,217 @@
...
+
+Last is the @code`ev/spawn` macro for concisely running a series of forms
```
Comment (pyrmont) [inline content/docs/event_loop.mdz:109]: This sounds unnatural.

File: content/docs/event_loop.mdz [hunk 6]
```diff
@@ -175,32 +224,44 @@ channel to monitor other tasks, handling errors or restarting them. This is a po
...
-  []
-  (print "starting new worker...")
-  (ev/go (fiber/new worker :tp) nil channel))
+ "Start a worker with the given channel as its supervisor."
+ []
+ (print "starting new worker...")
+ (ev/go (fiber/new worker :tp) nil channel))
 
-# make sure there are always two workers until counter gets to 100
+# make sure there are always two workers until the counter gets to 100
```
Comment (pyrmont) [inline content/docs/event_loop.mdz:241]: This is wrong in this context.

**Review by pyrmont — COMMENTED** [review]
This suggests to me that AI alignment theory is not something we need to be spending too much time on right now.

File: content/docs/event_loop.mdz (hunk 1 above)
Comment (pepe) [reply content/docs/event_loop.mdz:14]: It indeed is, but I think the biggest problem here is that I forget to remove "in"  before Janet. I know it is anthropomorphization I am guilty of. Or would you rather see me return the passive voice here?

File: content/docs/event_loop.mdz (hunk 4 above)
Comment (pepe) [reply content/docs/event_loop.mdz:107]: Good then? Can I resolve this?

File: content/docs/event_loop.mdz (hunk 5 above)
Comment (pepe) [reply content/docs/event_loop.mdz:109]: If you have any recommendations, that would be awesome.

**Review by sogaiu — COMMENTED** [review]
Needs more work to undo some of the grammarly changes.

**Comment by pepe** [comment]
It seems to me I went too far into Grammarly illusion. So I am closing it for some better future proposal.

**Pull request closed without merging**